util/files.py

```diff
@@ -46,7 +46,17 @@ def rm_path(path: Union[str, Path]) -> None:
         path.unlink()
 
 
-def is_subpath(child: str, parent: str):
+def is_subpath(child: PathLike, parent: Optional[PathLike] = None) -> bool:
+    """
+    If parent is not None, returns whether child is a subpath of parent.
+    If parent is None, check that child is a relative subpath i.e.
```

Comment:
relative subpath i.e.
Feels like the sentence is supposed to continue from there?